sending-a-simple-message-to-a-service/README.md

```diff
@@ -0,0 +1,325 @@
+# Sending a Simple Message to a Service
+
+## What you'll build
+
```

Comment:
```suggestion

```
Do we need to add a `Prerequisites` section?

Reply:
Need not remove any extra lines. +1 to cover the prerequisites in the `Set up the workspace` section.